### PR TITLE
Add Project Comment edit and delete actions

### PR DESCRIPTION
**Pull request by mateoclarke**
Closes https://github.com/cityofaustin/atd-data-tech/issues/4376

Backend changes (adding a status id to the comment so we can delete), run locally or in test environment

Permissions around comments in another issue: https://github.com/cityofaustin/atd-data-tech/issues/6468

### CODE REVIEW

**Review by sergiogcx — APPROVED** [review]
Sweet! This is looking really good! 👍  🗒️  💯

File: moped-editor/src/views/projects/projectView/ProjectComments.js [hunk 1]
```diff
@@ -100,6 +77,25 @@ const ProjectComments = () => {
...
+      setNoteText("");
+      refetch();
+      setCommentAddSuccess(true);
+      setEditingComment(false);
+      setTimeout(() => {
+        setCommentAddLoading(false);
+        setCommentAddSuccess(false);
+      }, 350);
+    },
+  });
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectComments.js:91]: I love seeing these here! 🥇

**Comment by amenity** [comment]
Looks good! I found a few small bugs (not specific to this PR) but nbd since this feature is in flux. I started an epic for future comments enhancements https://github.com/cityofaustin/atd-data-tech/issues/6496

**Review by amenity — APPROVED** [review]
LSTM!

**Pull request merged**